server/api/reply.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+const { Conversation, Reply, Activity } = require('../db');
+
+const RESULTS_PER_PAGE = 25;
```

Comment:
Is this being used yet? I wrote back end pagination middleware for grace-shopper we can utilize here that allowed changing results per page.

Reply:
sure yeah let's do that